CODE_OF_CONDUCT.md

```diff
@@ -285,12 +285,13 @@ is not in the scope of this Code of Conduct.
 #### **12.2.1** If the reportee is an employee of the Turing 
 
 The code of Conduct should be considered a norm-setting mechanism for the Open Research 
-Community Building Team before any involvement of HR in the interactions that take place within the team. 
+Community Management Team before any involvement of line managers or HR in the interactions that take place within the team. 
 
-If the reportee is a Turing employee, for instance, Community Managers and senior 
-researchers and the incident is considered out of the scope of this Code of Conduct 
-due to the severity of the matter/case, an HR process (formal or informal), will be started.
-In this case, the report will be passed to HR and/or appropriate personnel under the 
+If the reportee is a Turing employee, for instance, Senior Researcher, Senior Research Community Managers and Research Community Manager, and the incident is considered out of the scope by the report handling mechanism of this Code of Conduct due to the severity of the matter/case, the **employment policies will apply**.
+
+The first step for the community leaders handling the report should be to **escalate the issue to the line manager of the reportee**.
```

Comment:
Just to clarify for myself: Does this part only apply for cases where the CoC committee views the incident to be outside the scope? 

If it only applies to cases where the CoC made a decision and handled the case, the path we discussed today would be that the CoC committee would use their judgement on whether to inform the line manager of the person in question, and the line manager can then decide on what path to take formally with HR.

Reply:
Yes. The Committee should first handle the report and if they decide that this is serious enough to escalate to the line manager, only then they should -- folowing which [everything you said].